addons/statemachine/fnc_clockwork.sqf

```diff
@@ -18,7 +18,7 @@ SCRIPT(clockwork);
 
 {
     #ifdef STATEMACHINE_PERFORMANCE_COUNTERS
-    private _perfStartTime = diag_tickTime;
+        private _perfStartTime = diag_tickTime;
```

Comment:
`_perfStartTime` belongs to the same scope as `_stateMachine` and `_list` below and should be indented the same amount. Conditional preprocessing should not change the indentation level.